body.hbs

```diff
@@ -0,0 +1,40 @@
+<section id="abstract">
+  <p>
+    This is the most recent interoperability report for Vaccine Credentials using the <a href="https://w3c-ccg.github.io/vaccination-vocab/">Vaccine Certificate Vocabulary</a>.
+  </p>
+</section>
+<section id="sotd">
+</section>
+
+<section class="informative" id="introduction">
+  <h2>Introduction</h2>
+  <p>The purpose of this document is to demonstrate that there are multiple Issuers and Verifiers
+capable of processing Vaccine Credentials that conform to the <a href="https://w3c-ccg.github.io/vaccination-vocab/">Vaccine Certificate Vocabulary</a>.</p>
+  <section class="informative" id="testing-methodolgy">
+    <h3> Testing Methodology</h3>
+    <p>The testing framework for the Vaccine Credentials Data Model executes the following process:</p>
+    <ol class="algorithm">
+      <li>Takes Vaccination Credentials produced from WHO complaint ICD Codes</li>
+      <li>Passes these Vaccination Credentials to numerous Issuers</li>
+      <li>Asserts that the issuer has issued a valid Verifiable Credential</li>
+      <li>Passes the Vaccine Credential to numerous Verifiers</li>
+      <li>Collates the results in an interoperability matrix</li>
+    </ol>
+  </section>
+</section>
+
+<section class="informative" id="conformance-testing-results">
+  <h2> Conformance Testing Results </h2>
+  <section class="informative" id="Key">
+    <h3>Key</h3>
+    <ul>
+      <li>✓ - Success</li>
+      <li>❌ - Error</li>
+      <li>no support - Implementation explicitly does not support this feature</li>
+      <li>untested - This test is new, implementation has not been run on it yet</li>
+      <li>Implementations listed on the left side of the matrices are issuers.</li>
+      <li>Implementations listed on the top of the matrices are verifiers.</li>
+    </ul>
+  </section>
+  The results of the tests are show below:
+</section>
```

Comment:
Each vaxcert section doesn't have the appropriate `<section>` block and so isn't showing up in the table of contents. We want each certificate showing up in the table of contents.

We also want example output files for each test so people can see what these documents look like when they're signed. When the test suite is running, we want at least one of the outputs to be written in a way that it can be shown in the report. We want JSON-LD output and CBOR-LD output (so will need to be able to generate images next).

I suggest we put these files in the `results` directory and name them like the files in the `certificates` directory. For example: `XM1CE0.jsonld` (for the Verifiable Credential) and `XM1CE0.png` (for the CBOR-LD QR Code)

Reply:
> WRT to the tests showing up in the sidebar how does this look?

Yes, looks good.

Reply:
Yes, that is coming along. A few more suggestions:

Changes to the table:

* Note that the first items across the top row are verifiers (put information in top left cell)
* Note that the first column down the rows are issuers (put information in top left cell)
* CSS tilt text 45 degrees across top columns

Changes to the QR Code display (add this text):

```
CBOR-LD QR Code 
xxx byte CBOR-LD payload 
Version NN (nnn x nnn matrix) QR Code (base32-alphanumeric)
```

For the JSON-LD display:

* Set to fixed width or rest of width or text wrapping... don't allow large text to flow off to the right.
* See if a smaller font looks better?

Reply:
+1 to labeling the table col/rows.  At first glance I didn't know which was which.

Reply:
![vaccine-report-4](https://user-images.githubusercontent.com/278280/115449882-401c9900-a1e9-11eb-98df-8612856ea0ce.png)

Row and Column Labels added

Reply:
The first cell looks weird to me and probably has some accessibility and semantic layout issues.  Could avoid the 40% width thing, and use colspan/rowspan instead. Would this be better?

![Screen Shot 2021-04-20 at 6 50 20 PM](https://user-images.githubusercontent.com/109587/115473386-a4e7eb80-a209-11eb-9eda-3864d6718bf1.png)

The following could be ported back to the template, and made to use classes vs inline style:
```html
<thead>
           <tr>
             <th rowspan="2">
                <span class="rowLabel">
                Issuer
                </span>
              </th>
              <th colspan="7" style="text-align: center">Verifier</th>
            </tr>
            <tr>
                  <th>Danube Tech</th>
                  <th>Digital Bazaar</th>
                  <th>Mattr Labs</th>
                  <th>Mavennet</th>
                  <th>mesur.io</th>
                  <th>Spherity</th>
                  <th>Transmute</th>
            </tr>
</thead>
```

If the angled column header idea is used, this probably wouldn't work out as well.

Reply:
The JSON-LD is rendered oddly.  The first `{` is at col 0, everything else is indented 10 spaces, then 2 spaces.  Results in a lot of empty space.  Might be able to just flush the `<pre>` to the left in the template?

Reply:
![vaccine-report-6](https://user-images.githubusercontent.com/278280/115582978-73b5fc80-a297-11eb-84ab-0bf0036d6c18.png)